api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3323,4 +3325,35 @@ public void mergePatients_shouldMoveAllObsWithSameHierarchy() throws Exception {
 		assertEquals(2, encounterService.getEncounter(57).getObsAtTopLevel(true).size());
 	}
 
+
+	/**
+	 * Gets the patient , then sees if it can get the patient identifier by the patient program as well
```

Comment:
The comma, over spaced

Gets the patient, then sees if it can get the patient identifier by the patient program as well